File: app/src/main/java/com/foodie/foodvisit/widget/FoodVisitWidgetRemoteService.java
package com.foodie.foodvisit.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

/**
 * Created by kprabhu on 11/13/17.
 */

public class FoodVisitWidgetRemoteService extends RemoteViewsService {
    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new FoodVisitRemoteViewFactory();
    }
}

File: app/src/main/java/com/mintanable/foodvisit/widget/FoodVisitWidgetRemoteService.kt
package com.mintanable.foodvisit.widget

import android.content.Intent
import android.widget.RemoteViewsService

class FoodVisitWidgetRemoteService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory =
        FoodVisitRemoteViewFactory(applicationContext)
}
